build.gradle

```diff
@@ -422,6 +422,13 @@ subprojects {
         "-Xlint:unused"
       ]
     }
+// Scala 2.13 removed some scalac options (and hence we are removing them here, also)
+    if ( versions.baseScala.startsWith('2.13') ) {
+      scalaCompileOptions.additionalParameters.removeAll( [
+        "-Xlint:by-name-right-associative",
+        "-Xlint:unsound-match"
```

Comment:
These are the two relevant PRs for these changes:

* https://github.com/scala/scala/pull/6502
* https://github.com/scala/scala/pull/5969

Instead of doing it like this, I'd do only add these params if baseScala is 2.11 or 2.12 (similar to `-Xlint:constant`). Future versions of Scala won't have them either so the suggested approach is more future proof. Also, worth including the PR references in a comment near the conditional code.

Reply:
Done, new commit pushed.